### ISSUE TITLE
Missing Safari stable results for 349d418380

### ISSUE THREAD

**Issue by foolip** [web-platform-tests/results-collection#626]
https://wpt.fyi/test-runs?sha=349d418380

Safari 12 results weren't submitted.

**Comment by jugglinmike** [comment]
This collection attempt did not complete due to a timeout in chunk 3 of 20. [That's the same segment which has been dogging Firefox and Chrome over the past few weeks.](https://github.com/web-platform-tests/results-collection/issues/625) I have diagnosed a problem with wptrunner itself, suggesting that we'll be able to resolve this across all browsers with a patch to WPT.

I'm hoping to have such a patch ready for review by the end of the day. In the mean time, I've manually triggered a retry for the missing data.
